Sources/MessagesDisplayDataSource.swift

```diff
@@ -26,7 +26,9 @@ import Foundation
 
 public protocol MessagesDisplayDataSource: class, MessagesDataSource {
 
-    func messageColorFor(_ message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) -> UIColor
+    func textColorFor(_ message: MessageType) -> UIColor
```

Comment:
I see this as the only customisation available. I would not add a `textColor` method for the top / bottom / header / footer labels and since they are `attributedText` it doesn't really make sense either.